pkgs/development/python-modules/cfn-lint/default.nix

```diff
@@ -64,8 +65,14 @@ buildPythonPackage rec {
     "cfnlint.transform"
   ];
 
-  checkInputs = [ pytestCheckHook mock pydot ];
```

Comment:
Please do not remove pytestCheckHook.

Reply:
If I override checkPhase with my own check phase (in order to wrap pytest with `faketime`), does pytestCheckHook still do anything?

Reply:
> If I override checkPhase with my own check phase (in order to wrap pytest with `faketime`), does pytestCheckHook still do anything?

No it does not but I would prefer if we keep pytestCheckHook and just patch or disable the test.

Reply:
I am not sure if it's possible just to disable the specific test that checks the date -- I think they all might. But I defer to the actual maintainers for that.

Reply:
You can disable a list of test passing a list of string to  `disabledTests`, you need to now the name of the tests you want to disable.

Reply:
@tfmoraes: right, I know.

it seems like there are two possible routes to fix this package: one is to disable the currently failing tests. The other is to run the whole test suite under faketime in such a way that it’s immune to the “impure” influence of the clock.

I propose the `faketime` method, since I  think it will be more maintainable and robust. But if you all prefer the `disabledTests` route I’m happy to defer to your experience.

Reply:
Just disable the failing tests. Tests which require network or date/time are usually just disabled because they don't function in nixos.

Reply:
Okay.